colabs/raytune/W&B + RayTune.ipynb

```diff
@@ -6,7 +6,8 @@
       "name": "W&B-RayTune.ipynb",
```

Comment:
<p>Love the comments on Lines 31 + 34!</p><p> </p><p>Lines 1-5: I would import <code>torch.optim</code> and <code>tune</code> at the top: those are "standard" imports. Then, in this cell, import only the special things we need for logging this example: the example itself and the <code>wandb</code> code.</p><p>Line 35: Put spaces between mathematical operators: <code>100 * (1 - acc)</code> . This is part of the<a href="https://www.python.org/dev/peps/pep-0008/" target="_blank"> PEP8 Python style</a>.</p><p>Line 36: There are extra spaces before and after <code>"Error Rate "</code> , which should be <code>"error_rate"</code> (no space at the end!) to match the style of the other metrics (e.g. <code>mean_accuracy</code>). The spacing here is also part of the<a href="https://www.python.org/dev/peps/pep-0008/" target="_blank"> PEP8 Python style</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>